tests/functional/services/policy_engine/test_vulnerability_scanner.py

```diff
@@ -0,0 +1,134 @@
+import json
+import logging
+
+import pytest
+
+import tests.functional.services.policy_engine.utils.api as policy_engine_api
+from tests.functional.services.utils import http_utils
+
+
+@pytest.mark.parametrize(
+    "image_digest",
+    [
+        "sha256:80a31c3ce2e99c3691c27ac3b1753163214494e9b2ca07bfdccf29a5cca2bfbe",  # alpine-test
+        "sha256:406413437f26223183d133ccc7186f24c827729e1b21adc7330dd43fcdc030b3",  # debian-test
+        "sha256:fe3ca35038008b0eac0fa4e686bd072c9430000ab7d7853001bde5f5b8ccf60c",  # centos-test
+    ],
+)
+class TestVulnerabilityScanner:
+    def test_image_load_schema(self, image_digest, ingress_image, ingress_jsonschema):
+        image_load_resp: http_utils.APIResponse = ingress_image(image_digest)
+        assert image_load_resp == http_utils.APIResponse(200)
+        is_valid: bool = ingress_jsonschema.is_valid(image_load_resp.body)
+        if not is_valid:
```

Comment:
I see this pattern throughout here, where you are 1) checking for some value being present, not present, of a certain length, etc., 2) printing the value, and then 3) doing the assertation. I think it would be neater to use the assert error message in python, formatting the message with the relevant value. That will both make the error message easier to read in the logs (because you can provide context around what failed) and save lines of code here.

Reply:
I didn't know about the assert error message thing. Thanks, will try that.

Reply:
logging is now removed as a dependency for the tests. thanks for the recommendation!